src/files/files.ts

```diff
@@ -105,20 +105,30 @@ async function handleZippedModelFiles<T extends Content>(
     fileFormat = 'arraybuffer'
   }
 
+  let baseDir = ''
+
   zip.forEach((filePath, file) => {
     if (
       !basename(filePath).startsWith('.') &&
-      basename(filePath) !== WEARABLE_MANIFEST &&
-      basename(filePath) !== BUILDER_MANIFEST &&
-      basename(filePath) !== SCENE_MANIFEST
+      ![WEARABLE_MANIFEST, BUILDER_MANIFEST, SCENE_MANIFEST].includes(
+        basename(filePath)
+      )
     ) {
       fileNames.push(filePath)
       promiseOfFileContents.push(file.async(fileFormat) as Promise<T>)
+    } else if (
+      [WEARABLE_MANIFEST, SCENE_MANIFEST].includes(basename(filePath))
```

Comment:
Could it happen for these two files to be in different directories?

Reply:
it shouldn't happen. both files have to be at the same level.

I will add an extra validation to ensure both files are in the same directory